plugin/src/main/groovy/com/bmuschko/gradle/tomcat/tasks/TomcatJasper.groovy

```diff
@@ -132,6 +143,19 @@ class TomcatJasper extends Tomcat {
             jasperAttributes['compiler'] = getCompiler()
         }
 
+        if(getJspFiles()) {
+            // process filecollection into a comma-separated list of relative paths from uriroot
+            StringBuilder fileList = new StringBuilder()
+            Path basePath = getUriroot().toPath()
+            getJspFiles().each({ file ->
```

Comment:
Why don't we just use `getJspFiles().join(',')` here?

Reply:
Updated